discovery-provider/src/queries/get_tracks.py

```diff
@@ -177,6 +177,21 @@ def _get_tracks(session, args):
             base_query = base_query.join(TrackWithAggregates.aggregate_track).order_by(
                 sort_fn(AggregateTrack.save_count)
             )
+    else:
+        # Return the user's pinned track first if there is no specified sort_method
```

Comment:
Fun & interesting release caveat- 

If we roll this change out first, old clients will probably show two artist picks. If we roll the client change out first, we will end up showing no artist pick for a bit until all discovery nodes in the ecosystem are updated (a fun twist in our decentralized context is that this happens over a ~1 week time horizon).

Two potential solutions:
- We add an API query param to let clients dictate whether they get this API result back and then at some point down the line, we remove that and make it the default
- We have this response gated on a version of discovery https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/src/queries/get_health.py#L50 and then have clients check the discovery version prior to deciding on whether to fetch the artist pick the old way or not.

I think I like the former better because it's safer, but up to you I think!

Reply:
Good point. An API query param sounds the cleanest/easiest; I'll add that in

Reply:
Resolved offline -- @michellebrier is right!